app/components/hero/hero.styl

```diff
@@ -1,15 +1,13 @@
 // @import '../../variables.css'
 #hero
   height 100vh
-  background-repeat: no-repeat;
-  background-position-x: 66%
-  background-position-y 88%
+  background-image: linear-gradient(119deg, #441595, rgba(89, 30, 192, 0.94) 22%, rgba(110, 43, 217, 0.75) 53%, rgba(131, 74, 223, 0.85) 79%, #9f61f7);
   .sup-block
-    height 100vh
-    background-repeat no-repeat
-    background-position-y: 145%;
-    background-position-x: 42vw;
-    background-size 70%
+    // height 100vh
```

Comment:
You can simply delete lines you don't need. If you will feel a need for these rows of code you always can get them from Git